hazelcast/src/test/java/com/hazelcast/map/EntryProcessorTest.java

```diff
@@ -265,9 +263,10 @@ public void testEntryProcessorDeleteWithPredicate() {
 
     @Test
     public void testEntryProcessorWithKey() {
+        Config cfg = getConfig();
```

Comment:
If you are not modifying the config, I would completely get rid of the line. It just adds noise.